### PR TITLE
adding ability to work with sqlite compiled with -DSQLITE_OMIT_DECLTYPE

### LINKED ISSUE #751: feature request: add feature flag to link with sqlite builded with -DSQLITE_OMIT_DECLTYPE
I have to link with sqlite3 that part of big C++ library,
and sqlite3 is build with `-DSQLITE_OMIT_DECLTYPE`,
with such explanation:

> Disable returning the declared column type from query (reduces memory usage)

Quick hack to comment out:
```
    pub fn column_decltype(&self, idx: usize) -> Option<&CStr> {
        unsafe {
            let decltype = ffi::sqlite3_column_decltype(self.ptr, idx as c_int);
            if decltype.is_null() {
                None
            } else {
                Some(CStr::from_ptr(decltype))
            }
        }
    }
```

and all usage of it, show that rusqlite "core" not depend on this feature,
so it is easy enough feature, but may be I missed something?

### PR DESCRIPTION
**Pull request by Dushistov**
fix #751

### CODE REVIEW

**Comment by Dushistov** [comment]
@thomcc 

May be I should add new feature to:
```
          args: '--features "bundled-full session buildtime_bindgen"'
```
into github actions/codecov ?

**Comment by thomcc** [comment]
Oh, no, you should add it to the bundled-full though!

File: Cargo.toml [hunk 1]
```diff
@@ -86,6 +87,7 @@ bundled-full = [
     "uuid",
     "vtab",
     "window",
+    "column_decltype",
```
Comment (thomcc) [inline Cargo.toml:90]: Nit: these are in alphabetical order.

**Comment by gwenn** [comment]
Thanks.

**Pull request merged**